root/client/src/components/table/timeslot.js

```diff
@@ -33,17 +36,31 @@ export const TimeSlot = ({ date, startTime, duration, setPollAvailabilities, pol
       ...pollAvailabilities,
     };
 
-    const array = newPollAvail[date].filter((time) => time !== startTime);
+    const timestamp = new Date(`${date}T${startTime}`);
+    const dbTime = Intl.DateTimeFormat('default', { hour: 'numeric', minute: 'numeric', second: 'numeric', hour12: false, timeZone: 'UTC' }).format(timestamp);
```

Comment:
since this code is used more than once, it might be worth making it a function to call instead

Reply:
Created a function for this, placed function in util folder, file labeled tablehelpers.js